docs/HOWTO_add_routing_tiles.MD

```diff
@@ -0,0 +1,62 @@
+HOWTO_Add_routing tiles for Wahoo Elemnt ROAM

```

Comment:
could we rename the file to HOWTO_ADD_ROUTING_TILES.md to make it identically to the other docs/ files?

Reply:
Thanks a lot for the great suggestion! Let's update the doc.